### PR TITLE
Fixed TODO in FrameMain. 

### PR DESCRIPTION
**Pull request by FridaTveit**
Changed catch exception to catch specific exceptions.

### CODE REVIEW

File: src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java [hunk 1]
```diff
@@ -277,20 +277,25 @@ public RecognizeThread(FrameMain parentFrame) {
...
             String recognizedText = "";
             this.parentFrame.recognitionLabel.setText("processing...");
             int index = this.parentFrame.selectedIndex;
             try {
                 recognizedText = Main.systemLogic.recognize(this.parentFrame.car, false);
-            } catch (Exception ex) { // TODO exception
-                this.parentFrame.recognitionLabel.setText("failed");
-                ex.printStackTrace();
-                return;
+                this.parentFrame.recognitionLabel.setText(recognizedText);
```
Comment (oskopek) [inline src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java]: Is there any reason why you moved these two UI calls into the try block? Other than the fact that you can then omit two `return` statements in the catch blocks. If not, I'd be more comfortable adding the two `return`s and moving this back out of the try block. It shouldn't really matter execution-wise, but it's cleaner to have as short try blocks as possible.

File: src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java [hunk 2]
```diff
@@ -277,20 +277,25 @@ public RecognizeThread(FrameMain parentFrame) {
             this.parentFrame = parentFrame;
         }
 
+        private void setFailedAndPrintStackTrace(Exception exception) {
+            this.parentFrame.recognitionLabel.setText("failed");
```
Comment (oskopek) [inline src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java]: Optional: Remove all the `this` calls, if you feel like it :) In the spirit of #41

Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java]: Will do!

File: src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java (hunk 1 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java]: No, it was just to omit the two return statements. Okay, will move them back :)

File: src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java (hunk 2 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java]: Done :)

File: src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java (hunk 1 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/gui/windows/FrameMain.java]: Done :)

**Pull request merged**